.github/CONTRIBUTING.md

```diff
@@ -858,13 +880,82 @@ The following points can help you improve the readability of your tests:
 * Keep your tests isolated (more on this here: [The good test is isolated and flat](#the-good-test-is-isolated-and-flat))
 * Follow the AAA pattern as described above
 
-Below there are two examples. Which one is easiest to read? 
+Below are two examples. Which one is easiest to read and understand? 
+
+The following where: 
```

Comment:
Not sure I understand this sentence/structure? 🤔

Reply:
Took a stab at restructuring it a bit: https://github.com/kirbydesign/designsystem/pull/1534/commits/c2ab9b6e11af37116c3ba220b3c51721ec094013